src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,183 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+    
+    public MyLinkedList() {
+        head = null;
```

Comment:
if you want you can remove initialization from the constructor because all these values are default